spinn_front_end_common/utility_models/data_speed_up_packet_gatherer_machine_vertex.py

```diff
@@ -222,8 +227,9 @@ def generate_data_specification(
         # locate the tag ID for our data and update with port
         iptags = tags.get_ip_tags_for_vertex(self)
         iptag = iptags[0]
-        iptag.port = self._connection.local_port
+        iptag.port = 10000
```

Comment:
im not sure if this is a issue. more a flag for discussion. if youve set the port to a fixed port, wont this interfere with multi isntance runs?

Reply:
I have added a comment to explain that this doesn't matter

Reply:
this might be just my first day rust. but i cant see the comment in the changes. also, if we're going to use a constant. can it not be a magic number?

Reply:
The comment is 2 lines up.  This number is "magic" by definition...

Reply:
yea, was just rust. sorry